src/mslice/presenters/slice_plotter_presenter.py

```diff
@@ -123,3 +124,10 @@ def _cache_intensity_correction_methods(self):
                                     self.show_dynamical_susceptibility_magnetic)
         IntensityCache.cache_method(cat, IntensityType.D2SIGMA, self.show_d2sigma)
         IntensityCache.cache_method(cat, IntensityType.SYMMETRISED, self.show_symmetrised)
+        IntensityCache.cache_method(cat, IntensityType.GDOS, self.show_gdos)
+        IntensityCache.cache_slice_type(IntensityType.SCATTERING_FUNCTION, "scattering_function")
+        IntensityCache.cache_slice_type(IntensityType.CHI, "chi")
+        IntensityCache.cache_slice_type(IntensityType.CHI_MAGNETIC, "chi_magnetic")
+        IntensityCache.cache_slice_type(IntensityType.D2SIGMA, "d2sigma")
+        IntensityCache.cache_slice_type(IntensityType.SYMMETRISED, "symmetrised")
+        IntensityCache.cache_slice_type(IntensityType.GDOS, "gdos")
```

Comment:
@MialLewis Ah yeah... so I don't actually know what the first six lines here  (lines 120-125) are meant to do... they were not used at all in the old code. I had thought to reuse them for what I want to do but they refer to the methods of `SlicePlotterPresenter` and what I wanted were the methods of `SlicePlot`.

Reply:
Yes I think removing them is for the best. I believe I put these in here for consistency with `cut_plotter_presenter` and for future use, but this ends up being confusing.